components/TButtonShare.vue

```diff
@@ -156,8 +156,15 @@ export default {
       this.$nuxt.$loading.start()
 
       try {
+        const { account } = useAuth()
         const result = await axios.get(
-          `https://us-central1-wedance-4abe3.cloudfunctions.net/hooks/share${this.$route.path}?timezone=Europe/Berlin`
+          `https://us-central1-wedance-4abe3.cloudfunctions.net/hooks/share${
+            this.$route.path
+          }?timezone=${
+            !account.value?.zone
+              ? Intl.DateTimeFormat().resolvedOptions().timeZone
```

Comment:
I would rather ensure that account.zone is set in auth module and here just use it's value. There gonna be more places like that later